consensus/state.go

```diff
@@ -712,8 +712,8 @@ func (cs *ConsensusState) handleMsg(mi msgInfo) {
 	if err != nil {
 		// Causes TestReactorValidatorSetChanges to timeout
 		// https://github.com/tendermint/tendermint/issues/3406
-		// cs.Logger.Error("Error with msg", "height", cs.Height, "round", cs.Round,
-		// 	"peer", peerID, "err", err, "msg", msg)
+		cs.Logger.Error("Error with msg", "height", cs.Height, "round", cs.Round,
```

Comment:
i could not replicate the issue here over multiple tests, if we want to leave it commented, i suggest we comment out the if statement

Reply:
Okay, maybe it's OK to merge then

Reply:
I would like to spin up a server and test it on a better machine before we merge, will do so tonight

Reply:
```
^Csignal: interrupt
FAIL	github.com/tendermint/tendermint/consensus	182.613s
PASS
--- FAIL: TestReactorRecordsVotesAndBlockParts (0.23s)
    reactor_test.go:308:
        	Error Trace:	reactor_test.go:308
        	Error:      	Not equal:
        	            	expected: true
        	            	actual  : false
        	Test:       	TestReactorRecordsVotesAndBlockParts
        	Messages:   	number of votes sent should have increased
    reactor_test.go:309:
        	Error Trace:	reactor_test.go:309
        	Error:      	Not equal:
        	            	expected: true
        	            	actual  : false
        	Test:       	TestReactorRecordsVotesAndBlockParts
        	Messages:   	number of votes sent should have increased
FAIL
exit status 1
FAIL	github.com/tendermint/tendermint/consensus	19.760s
^Csignal: interrupt
FAIL	github.com/tendermint/tendermint/consensus	262.879s
^Csignal: interrupt
FAIL	github.com/tendermint/tendermint/consensus	195.406s
```

ran the tests 50 times on a server and these issues arose, now testing both master and this branch, as of now seems the long ones seem to happen on master as well

@melekes is this normal?

Reply:
so it seems that the consensus reactor is touchy, I wasnt able to recreate the test failure you had, but maybe for now we leave it commented like before and once the reactor is refactored, we can look at this again.

Reply:
> @melekes is this normal?

no